packages/client/types/constEnums.ts

```diff
@@ -307,14 +307,6 @@ export const enum SubscriptionChannel {
   ORGANIZATION = 'organization'
 }
 
-export const enum SuggestedActionTypeEnum {
```

Comment:
we now have a single source of truth for these in the graphql enum

Reply:
yeah, single source of truth! the `keyof typeof` is really helpful when doing TypeScript legacy support.
btw: have we tried any migration tool like [ts-migrate](https://github.com/airbnb/ts-migrate) before? I'm curious about how it will actually work.

Reply:
the last time i checked it out it was still a little buggy & at best it would just explicitly cast things as `any` as a fallback.
this little exercise probably took the team 30-40 hours to complete, but we uncovered some cool bugs & got some really useful patterns (like keyof typeof) in place so future developers have some good patterns to go off of.